src/shared/components/ProgressBarKey/ProgressBarKey.tsx

```
@@ -0,0 +1,51 @@
+
+  return (
+    <div
+      className={styles.container}
+      role="progressbar"
+      aria-valuemin={1}
+      aria-valuemax={4}
+      aria-valuenow={normalizedStep}
+      aria-label={`진행 상황: ${normalizedStep}단계 (총 4단계)`}
+    >
+      <div className={styles.track} />
+      <div className={styles.stepsWrapper}>
```

Biome (1.9.4)

[error] The HTML element with the interactive role "progressbar" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "progressbar".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)